farmdata2_modules/fd2_tabs/fd2_example/ui.html

```diff
@@ -176,11 +213,56 @@
 
         waitingOnLogs: false,
         seedingLogs: [],
+
+        regularExp: null,
+        selectedRegOpt: null,
+        regexInputVal: null,
+        regexMatchVal: false,
+        disabledMap: {
+            "DropdownWithAllComponent": false,
+            "RegexInputComponent": false,
+            "DateSelectionComponent": false,
+            "DateRangeSelectionComponent": false,
+        }
     },
     methods: {
+        checkDisabled(key) {
+            if(this.disabledMap[key]) {
+                return "Enable"
+            } else {
+                return "Disable"
+            }
+        },
+        makeDisabled(key) {
```

Comment:
This function name doesn't seem to accurately describe the function's purpose.  Maybe "toggleDisabled"?

Reply:
Changed function name as suggested.